openwisp_controller/config/api/serializers.py

```diff
@@ -48,6 +48,14 @@ class Meta(BaseMeta):
             'modified',
         ]
 
+    def validate_organization(self, value):
```

Comment:
This solution looks good to me, however, would be even better if we avoided showing the empty org option altogether, if that's doable and it's enforced, that is even better (simpelr and better UX). Is that doable? We do this for the admin, I suggest check the code in openwisp-users to see how we do it: https://github.com/openwisp/openwisp-users/blob/aea15b42085967d16dfb82dd19c1bc8d261209b2/openwisp_users/multitenancy.py#L70-L78

I am sure we need it also for VPNs. I created a staff non super user with the "administrator" role and I see that they can create a VPN server in the admin and the option to select "shared systemwide" is not present.
I suggest proceeding with creating a reusable solution, first implement it here in this PR, then we'll port it to openwisp-users and update this PR.

Reply:
> however, would be even better if we avoided showing the empty org option altogether,
Is that doable?

Yes It is doable, by just altering the queryset in the `FilterSerializerByOrganization` class, we can exclude the empty field from the option for non-super users, but after implementing it there, we will not be required to add this field level validation, as it will not get triggered.

Reply:
sounds good!